.circleci/config.yml

```diff
@@ -1,12 +1,11 @@
 version: 2.1
 
 references:
-  circleci_docker_primary: &circleci_docker_primary trussworks/circleci-docker-primary:822fac1c30f3bb7d5d595bed5d2dc86265c4f2f0
-
+  circleci_docker_primary: &circleci trussworks/circleci:6986bb9022e5a83599feb66a7128a2d0fa12732a
```

Comment:
oh wait. you need to change `circleci_docker_primary` to `circleci`

Reply:
oh yeah just gunna go and change that.....and there we go